.github/workflows/codeql.yml

```diff
@@ -0,0 +1,34 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches:
+      - master
+      - "!renovate/**"
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches:
+      - master
+  schedule:
+    - cron: "0 0 * * 0"
```

Comment:
Huh - I wonder what the value of doing this on a cron will be?  During a PR someone can capture and react to it, but on a cron I'm not sure how it would lead to someone noticing.

Reply:
You will see the number of issues on the repository's security tab. I can remove it if you don't want this.

Reply:
Nah, just learning :)